src/Microsoft.PowerShell.Commands.Utility/commands/utility/Write-Object.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.PowerShell.Commands
 {
     #region WriteOutputCommand
     /// <summary>
-    /// This class implements Write-output command
+    /// This class implements Write-output command.
```

Comment:
Suggest inserting 'the' before Write-Output.
Fix casing for Write-Output

Reply:
Fixed.